### DISCUSSION TITLE
docker build image is related to the disk of the local machine?

### DISCUSSION THREAD

**Discussion by z7658329** [moby/moby#48215] [Q&A] [answered]
<img width="419" alt="企业微信截图_62709e8f-ce0d-4f6c-a9ad-cc100b63758a" src="https://github.com/user-attachments/assets/d3be18e1-c232-4b83-8abb-e28a717a352b">

it takes about 10 mins to execute step 1 to step 8, why costs so much time? I want to ask if the docker build image is related to the disk of the local machine?

**Comment by thaJeztah** [comment, marked as answer]
Looks like this is a duplicate of a ticket you also opened, and in which some answers were provided;

- https://github.com/moby/moby/issues/48229
